src/IFX007T-Motor-Control.cpp

```diff
@@ -659,7 +895,7 @@ void IFX007TMotorControl::setADCspeedFast(void)
 
 void IFX007TMotorControl::setPwmFrequency(uint8_t pin, uint16_t divisor)
 {
-  setAnalogWriteFrequency(pin, 15000);       // 31250 Hz, obviously we nee the half (fault in XMCforArduino?)
+  setAnalogWriteFrequency(pin, 30000);       // 31250 Hz, obviously we nee the half (fault in XMCforArduino?)
```

Comment:
Ja habe ich auch schon festgestellt, dass da was nicht passt. Ich denke das ist irgendein Pre-Scaler nicht richtig eingestellt. Ist mal wieder ein XMC-For-Arduino Fehler :)

Reply:
Ja das ist mal so und mal so